src/einsteinpy/utils/__init__.py

```diff
@@ -1,4 +1,8 @@
 from .bl_coord_transforms import *
 from .coord_transforms import *
 from .scalar_factor import scalar_factor, scalar_factor_derivative
-from .schwarzschild_utils import schwarzschild_radius, time_velocity
+from .schwarzschild_utils import (
+    schwarzschild_radius,
+    time_velocity,
+    schwarzschild_radius_dimensionless,
```

Comment:
```
from .schwarzschild_utils import (
 .
 .
 .
)  # isort:skip
```
Black and isort are conflicting. This would force isort to ignore the line

Reply:
Slight edit
```
from .schwarzschild_utils import (  # isort:skip
 .
 .
 .
)  # isort:skip
```
And then run `black` and `isort` again 😃

Reply:
i dont understand i did isort and black but it is still showing failed.